defaults/main.yaml

```
@@ -2,8 +2,17 @@
 mariadb:
   prefix:
     config: /usr/local/etc
+  config:
+    filename: my.cnf
   databases: {}
   database_defaults:
     encoding: utf8mb4
     collation: utf8mb4_unicode_ci
+  socket:
   my.cnf: {}
+  galera:
+    cluster: False
```

Please use `yes`/`no` in YAML.
```suggestion
    cluster: no
```

---

Doesn't this get inconsistent throughout the project if there are different ways of writing the same information?